components/lane_follower/src/individual_follower.py

```diff
@@ -0,0 +1,150 @@
+import math
+
+import cv2
+import numpy as np
+import toml
+from einops import repeat
+
+
+class ifResult:
+    def __init__(self):
+        self.result_img = None
+        self.empty_windows = 0
+        self.heading = 0
+        self.slope = 0
+
+    pass
+
+
+class IndividualFollower:
+
+    def __init__(self, toml_path='Follower_Consts.toml'):
+
+        self.consts = toml.load(toml_path)["Individual_Follower"]
+        # self.fit is set in Plot_Line
+        self.fit = None
+        # These two are set below
+        self._binary_warped = None
+        self._histogram = None
+
+    # Need to determine if these should remain properties
+    # Or if they should be passed as arguments to Plot_Line
+    @property
+    def _binary_warped(self, value: np.ndarray):
```

Comment:
`@property` should only get used for public members and without arguments so that they just act as a variable access.